frontend/modules/recipe/components/Recipe.js

```diff
@@ -213,6 +208,22 @@ var RecipeScheme = injectIntl(React.createClass({
     );
   },
 
+  showRecipeImage: function() {
+    if (this.props.data.photo) {
+      return (
+        <img className="banner-img img-responsive hidden-xs" src={ this.props.data.photo } />
+      );
+    }
+  },
+
+  showRecipeImageThumb: function() {
+    if (this.props.data.photo_thumbnail) {
+      return (
+        <img className="img-responsive" src={ this.props.data.photo_thumbnail } />
```

Comment:
I think we should also show the default image here as well. Just doesn't look as good without it, IMO.